tests/integrations/test_keras.py

```diff
@@ -901,16 +919,19 @@ def test_data_logger_pred_inferred_proc_no_classes(test_settings, live_mock_serv
                 row[tcols.index("3dimages:image")], wandb.data_types.Image
             )
             assert isinstance(row[tcols.index("video:video")], wandb.data_types.Video)
+        run.finish()
 
 
 def test_keras_dsviz(dummy_model, dummy_data, runner, live_mock_server, test_settings):
-    with wandb.init(settings=test_settings) as run:
-        dummy_model.fit(
-            *dummy_data,
-            epochs=2,
-            batch_size=36,
-            validation_data=dummy_data,
-            callbacks=[WandbCallback(log_evaluation=True)]
-        )
-        assert wandb.run.summary["validation_predictions"] is not None
-        assert wandb.run.summary["validation_predictions"]["artifact_path"] is not None
+    run = wandb.init(settings=test_settings)
+    dummy_model.fit(
+        *dummy_data,
+        epochs=2,
+        batch_size=36,
+        validation_data=dummy_data,
+        callbacks=[WandbCallback(log_evaluation=True)]
+    )
+    assert wandb.run.summary["validation_predictions"] is not None
+    assert wandb.run.summary["validation_predictions"]["artifact_path"] is not None
+    assert wandb.run.summary["validation_predictions"]["_type"] == "table-file"
```

Comment:
Nice!

Reply:
Actually, I am not sure what happened here. It used to be `artifact_type` and not `_type`. This way, the test passes. @raubitsj do you have any intel on this?